ntbs-service/Pages/Notifications/LinkedNotifications.cshtml.cs

```diff
@@ -21,16 +22,18 @@ public async Task<IActionResult> OnGetAsync(int id)
                 return NotFound();
             }
 
-            await AuthorizeAndSetBannerAsync();
-
-            NotificationId = Notification.NotificationId;
-            await GetLinkedNotifications();
-
-            if (Group == null)
+            try
+            {
+                await TryGetLinkedNotifications();

```

Comment:
I think `Try` in the name suggests that it is NOT going to throw an exception if it fails.
[this seems to confirm that as a spread pattern in c#](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/exceptions-and-performance#try-parse-pattern)
Can we either change this to return a boolean, OR rename to remove "Try" from the name? (I *think* the latter should be fine performance wise)